examples/fluids/qfunctions/newtonian.h

```diff
@@ -710,69 +461,57 @@ CEED_QFUNCTION(IJacobian_Newtonian)(void *ctx, CeedInt Q,
 
     // Total flux
     CeedScalar dFlux[5][3];
-    for (int j=0; j<3; j++) {
-      dFlux[0][j] = dF_inviscid[j].density;
-      for (int k=0; k<3; k++)
-        dFlux[k+1][j] = dF_inviscid[j].momentum[k] - dstress[k][j];
-      dFlux[4][j] = dF_inviscid[j].E_total + dFe[j];
-    }
+    FluxTotal(dF_inviscid, dstress, dFe, dFlux);
 
-    for (int j=0; j<3; j++) {
-      for (int k=0; k<5; k++) {
+    for (int j=0; j<3; j++)
+      for (int k=0; k<5; k++)
         Grad_v[j][k][i] = -wdetJ * (dXdx[j][0] * dFlux[k][0] +
                                     dXdx[j][1] * dFlux[k][1] +
                                     dXdx[j][2] * dFlux[k][2]);
-      }
-    }
 
     const CeedScalar dbody_force[5] = {0, ds.U.density *g[0], ds.U.density *g[1], ds.U.density *g[2], 0};
     for (int j=0; j<5; j++)
       v[j][i] = wdetJ * (context->ijacobian_time_shift * dU[j] - dbody_force[j]);
 
-    if (1) {
```

Comment:
I know I had asked @jedbrown about removing this before (see [here](https://github.com/CEED/libCEED/pull/962#discussion_r889586560)). Not sure if the same reasoning applies now though. Jed mentions refactoring this block of code, so maybe this is good to go now.

Reply:
:shrug: Not important to me any more, as our stabilization code has been exercised more.